olive/platform_sdk/qualcomm/env.py

```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import os
+import platform
+from pathlib import Path
+
+from olive.platform_sdk.qualcomm.constants import SDKTargetDevice
+
+
+class SDKEnv:
+    def __init__(self, sdk: str, root_env_name: str, target_arch: str = None, dev: bool = False):
+        self.sdk = sdk
+        self.root_env_name = root_env_name
+        # dev: Whether to use the development version of the SDK,
+        # only applicable to SNPE/QNN x86_64 linux/windows
+        self.dev = dev
+        self.target_arch = self._verify_target_arch(target_arch) if target_arch else self._infer_target_arch()
+        self.delimiter = os.path.pathsep
+
+    @property
+    def sdk_root_path(self) -> str:
```

Comment:
>sdk_root_path [](http://example.com/codeflow?start=8&length=13)

need we save it as attribute in __init__?

Reply:
Updated.